library/src/scala/tasty/reflect/Printers.scala

```diff
@@ -1445,6 +1445,11 @@ trait Printers
         case Type.SymRef(sym, prefix) =>
           prefix match {
             case Types.EmptyPrefix() =>
+              if (sym.owner.flags.is(Flags.Package)) {
+                val packagePath = sym.owner.fullName.stripPrefix("<root>").stripPrefix("<empty>").stripPrefix(".")
+                if (packagePath != "")
+                  this += packagePath += "."
+              }
```

Comment:
I'm wondering inside the `if`, what is the value of `prefix` at runtime?

Reply:
It is a `Types.EmptyPrefix()` as the pattern matching indicates.

Reply:
If that is the case, it seems to me the type is ill-formed, the `prefix` is wrong -- or I misunderstood the invariant for `TermRef`. We can get this in to unblock other issues, but it would be nice to clarify the invariant about `TermRef`. /cc: @odersky .